src/js/libcs/Operators.js

```diff
@@ -4530,3 +4530,68 @@ evaluator.dropped$0 = function() {
 evaluator.droppoint$0 = function() {
     return dropPoint;
 };
+
+evaluator.parsecsv$1 = function(args, modifs) {
+    // more strict float parsing
+    var filterFloat = function(value) {
+        if (/^(\-|\+)?([0-9]+(\.[0-9]+)?|Infinity)$/
+            .test(value))
+            return Number(value);
+        return NaN;
+    };
+
+    // autoconvert like tokenize
+    var aucon = true;
+    var mcon = evaluateAndVal(modifs.autoconvert);
+    if (mcon.ctype === "boolean") aucon = mcon.value;
+
+    var delim = "";
+    var md = modifs.delimiter;
+    if (md && md.ctype === "string" && md.value.length === 1) delim = md.value;
+
+    var nl = "";
+    var mnl = modifs.newline;
+    if (mnl && mnl.ctype === "string" && mnl.value.length === 2) nl = mnl.value;
+
+    var papaconfig = {
+        delimiter: delim,
+        newline: nl
+    };
+
+    var str = evaluateAndVal(args[0]).value;
+    var parsed = window.Papa.parse(str, papaconfig);
```

Comment:
This will throw an exception if `Papa` is not available, e.g. because the `parsecsv` call was not part of one of the standard scripts but instead used the script of a button or passed to the widget via `evokeCS`. Unlikely scenarios, but in my opinion not unlikely enough to avoid handling them since exceptions can have pretty drastic effects in CindyScript evaluation.